src/thing.rs

```diff
@@ -21,54 +32,92 @@ pub struct Thing {
     // The context can be arbitrarily complex
     // https://www.w3.org/TR/json-ld11/#the-context
     // Let's take a value for now and assume we'll use the json-ld crate later
+    /// A [JSON-LD @context](https://www.w3.org/TR/json-ld11/#the-context)
     #[serde(rename = "@context", default = "Thing::default_context")]
     pub context: Value,
 
+    /// A unique identifier
     pub id: Option<String>,
 
+    /// JSON-LD semantic keywords
     #[serde(rename = "@type", default)]
     #[serde_as(as = "Option<OneOrMany<_>>")]
     pub attype: Option<Vec<String>>,
 
+    /// Human-readable title to be displayed
     pub title: String,
 
+    /// Multi-language translations of the title
     pub titles: Option<MultiLanguage>,
 
+    /// Human-readable additional information
     pub description: Option<String>,
 
+    /// Multi-language translations of the description
     pub descriptions: Option<MultiLanguage>,
 
+    /// Version information
     pub version: Option<VersionInfo>,
 
+    /// Time of creation of this description
+    ///
+    /// It may be used for caching purposes.
     #[serde(with = "time::serde::rfc3339::option", default)]
     pub created: Option<OffsetDateTime>,
 
+    /// Time of last update of this description
+    ///
+    /// It may be used for caching purposes.
     #[serde(with = "time::serde::rfc3339::option", default)]
     pub modified: Option<OffsetDateTime>,
 
+    /// URI to the device maintainer
+    ///
+    /// To be used to ask for support.
     // FIXME: use AnyURI
     pub support: Option<String>,
 
+    /// Base URI to be used to resolve all the other relative URIs
+    ///
+    /// NOTE: the JSON-LD @context is excluded.
     // FIXME: use AnyURI
     pub base: Option<String>,
 
+    /// Property-based [Interaction Affordances]
+    ///
```

Comment:
The last empty line can be removed (did github remove the "suggested change" button?)